lib/AppInfo/Application.php

```diff
@@ -1,56 +0,0 @@
-<?php
-/**
- * @author Björn Schießle <schiessle@owncloud.com>
- *
- * @copyright Copyright (c) 2015, ownCloud, Inc.
- * @license AGPL-3.0
- *
- * This code is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License, version 3,
- * as published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License, version 3,
- * along with this program.  If not, see <http://www.gnu.org/licenses/>
- *
- */
-
-
-namespace OCA\FilesTextEditor\AppInfo;
-
-use OCA\FilesTextEditor\Controller\FileHandlingController;
-use OCP\AppFramework\App;
-use OCP\AppFramework\IAppContainer;
-
-class Application extends App {
-
-	/**
-	 * @param array $urlParams
-	 */
-	public function __construct(array $urlParams = array()) {
-		parent::__construct('files_texteditor', $urlParams);
-
-		$container = $this->getContainer();
-		$server = $container->getServer();
-
-		$container->registerService('FileHandlingController', function (IAppContainer $c) use ($server) {
-			$user = $server->getUserSession()->getUser();
-			if ($user) {
-				$uid = $user->getUID();
-			} else {
-				throw new \BadMethodCallException('no user logged in');
-			}
-			return new FileHandlingController(
-				$c->getAppName(),
-				$server->getRequest(),
-				$server->getL10N($c->getAppName()),
-				$server->getLogger(),
-				$server->getUserFolder($uid)
```

Comment:
And the magic recipe is

* :heart: 
* :rabbit: 
* :cat: 
* :sparkles: 

Mix it. And it works....